worlds/dark_souls_3/Locations.py

```diff
@@ -688,6 +688,269 @@ def get_name_to_id() -> dict:
+        "IBV: Ring of Favor", # Sulyvahn's Beast Duo drop
+        "IBV: Aldrich Faithful", # Sulyvahn's Beast Duo reward
+        "AL: Aldrich's Ruby", # Deep Accursed drop
+        "PC: Logan's Scroll", # NPC drop
+        "PC: Eleonora", # Monstrosity of Sin drop
+        "LC: Irithyll Rapier", # Boreal Outrider drop
+        "GA: Crystal Scroll", # Crystal Sage drop
+        "GA: Outrider Knight Helm",
+        "GA: Outrider Knight Armor",
+        "GA: Outrider Knight Gauntlets",
+        "GA: Outrider Knight Leggings",
+        "GA: Golden Wing Crest Shield", # NPC drop
+        "GA: Onikiri and Ubadachi", # NPC drop
+        "GA: Sage's Crystal Staff", # NPC drop
+        "AP: Dragon Tooth", # NPC drop
+        "DH: Flame Fan", # Desert Pyromancer Zoey drop
+        "RC: Iron Dragonslayer Helm",
+        "RC: Iron Dragonslayer Armor",
+        "RC: Iron Dragonslayer Gauntlets",
+        "RC: Iron Dragonslayer Leggings",
+        "RC: Crucifix of the Mad King", # Shira drop
+    ]),
+
+    "Key Locations": frozenset([
+        "HWL: Cell Key",
+        "ID: Jailer's Key King",
+        "ID: Jailbreaker's Key",
+        "ID: Old Cell Key",
+        "LC: Grand Archives Key",
+        "PW: Contraption Key",
+
+        # Not currently randomized:
+        # "FS: Grave Key",
+        # "FS: Lift Chamber Key",
+        # "FS: Tower Key",
+    ]),
+
+    "Guarded by Keys": frozenset([
+        # Guarded by Tower Key
+        "FSBT: Covetous Silver Serpent Ring",
+        "FSBT: Fire Keeper Robe",
+        "FSBT: Fire Keeper Gloves",
+        "FSBT: Fire Keeper Skirt",
+        "FSBT: Estus Ring",
+        "FSBT: Fire Keeper Soul",
+
+        # Guarded by Cell Key
+        "HWL: Greirat's Ashes",
+
+        # Guarded by Jailer's Key Ring
+        "ID: Prisoner Chief's Ashes",
+        "ID: Karla's Ashes",
+        "ID: Karla's Pointed Hat",
+        "ID: Karla's Coat",
+        "ID: Karla's Gloves",
+        "ID: Karla's Trousers",
+
+        # Guarded by Jailbreaker's Key
+        "ID: Bellowing Dragoncrest Ring",
+
+        # Guarded by Old Cell Key
+        "ID: Covetous Gold Serpent Ring",
+    ]),
+
+    # Locations that are particularly tricky to find or get to, for players
+    # without an encyclopedic knowledge of DS3 who don't want to get stuck
+    # looking for an invisible wall or one random mob with a guaranteed drop.
+    "Hidden Locations": frozenset([
+        # TODO: this list isn't at all comprehensive, expand it
+
+        "DC: Dragonhead Shield", # requires gesture
+        "SL: Speckled Stoneplate Ring", # requires careful ballista shot
+
+        # Out-of-the-way cave
+        "FK: Golden Scroll",
+        "FK: Antiquated Dress",
+        "FK: Antiquated Gloves",
+        "FK: Antiquated Skirt",
+
+        # In lava
+        "SL: Toxic Mist",
+        "SL: White Hair Talisman",
+        "SL: Sacred Flame",
+
+        # Guaranteed drop from normalish enemy
+        "CD: Deep Ring",
+        "PC: Eleonora",
```

Comment:
I understand both decisions, but putting Eleonora on both Miniboss Rewards and Hidden Locations feels odd.